src/pages/ListPage.js

```diff
@@ -1,12 +1,13 @@
 import styled from 'styled-components';
 import Card from '../components/Card';
-import FormSubmitMessage from '../components/FormSubmitMessage';
+import FormSubmitMessage from '../components/FormSubmitMessage.js';
+import EmptyListMessage from '../components/EmptyListMessage.js';
 
 export default function ListPage({ notes, isFormSubmitted }) {
   return (
     <StyledPage>
-      {isFormSubmitted ? <FormSubmitMessage>Your data has been submitted!</FormSubmitMessage> : ''}
-
+      {notes.length === 0 ? <EmptyListMessage /> : ''}
+      {isFormSubmitted ? <FormSubmitMessage /> : ''}
```

Comment:
Here, you could simply replace `FormSubmitMessage` with its content (the styled `p` element).